core-web/libs/dotcms-scss/angular/dotcms-theme/components/messages/_toast.scss

```diff
@@ -0,0 +1,122 @@
+@use "variables" as *;
+@import "mixins";
+
+$toast-bottom-border: 8px solid;
+
+.p-component.p-toast p-toastitem .p-toast-message {
+    box-shadow: $shadow-m;
+    background-color: $white;
+    width: 23.5rem;
+    height: fit-content;
+    border-radius: $border-radius-md;
+
+    .p-toast-message-content {
+        padding: $spacing-3;
+        padding-bottom: $spacing-4;
+        display: flex;
+        align-items: flex-start;
+        gap: $spacing-1;
+        border-radius: $border-radius-md;
+
+        .p-toast-message-icon {
+            height: 1.375rem;
```

Comment:
Is not squared?

Reply:
it is, but i need it to be that height to center it with the text

Reply:
But if the text is multiples lines, we should just add a top padding prob?

Reply:
Not sure how to do it. I was testing multiline and it looks like every line has a height of 22.4px

Reply:
Not sure how to do it. I was testing multiline and it looks like every line has a height of 22.4px